src/Checker/DeadCode/DeadCodeFinder.php

```diff
@@ -113,8 +137,8 @@ protected function getAllExtendedCoreClasses(string $path): array
                 continue;
             }
 
-            preg_match('/namespace (?<namespace>\S*);\\n/', $fileContent, $matchesNamespace);
-            preg_match('/class (?<class>\S*) /', $fileContent, $matchesClass);
+            preg_match('/^namespace (?<namespace>\S*);/m', $fileContent, $matchesNamespace);
+            preg_match('/^class (?<class>\S*) /m', $fileContent, $matchesClass);
```

Comment:
the class can be abstract. reuse this one `preg_match('/(?<fullClass>\\n(?<abstract>(abstract )?)class (?<class>\S*) extends (?<extendedClass>\S+))/', $fileContent, $matches);`